app/main.py

```diff
@@ -2,24 +2,82 @@
 
 
 class IntegerRange:
-    pass
+
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+        self.private_name = None
+
+    def __set_name__(self, owner: str, name: str) -> None:
+        self.private_name = f"_{name}"
+
+    def __get__(self, instance: str, owner: None = None) -> int:
+        if instance is None:
+            return self
+
+        return getattr(instance, self.private_name, None)
```

Comment:
The `__get__` method should return the value of the attribute from the instance, not `None`. It should return an `int` type, which represents the current value of the attribute it is managing. Consider using `getattr(instance, self.private_name)` without the default `None` to ensure it returns the correct value.